etc/umcg-atd.cfg

```diff
@@ -42,30 +42,30 @@ declare -a NOTIFICATION_ORDER_PHASE_WITH_STATE=(
 )
 
 declare -A NOTIFY_FOR_PHASE_WITH_STATE=(
-	['moveAndCheckSamplesheets:failed']='email'
+	['moveAndCheckSamplesheets:failed']='channel'
```

Comment:
what's up with channel?

Reply:
It instructs the notification.sh script to send a massage to an MS Teams channel instead of sending an email.